hera_xblocks/hera_pages/hera_pages/static/html/hera_pages.html

```diff
@@ -2,22 +2,94 @@
+                                                                <td>
+                                                                    <%
+                                                                        try:
+                                                                            column_answer = user_answers[index][user_answer_input_index] if user_answers else ''
+                                                                        except IndexError:
+                                                                            column_answer = ''
+                                                                    %>
+                                                                    <input type="text" name="${index}" value="${column_answer}"/>
+                                                                </td>
+                                                                <% user_answer_input_index += 1 %>
+                                                            % else:
+                                                                <td>${column['value']}</td>
+                                                            % endif
+                                                        % endfor
+                                                    </tr>
+                                                % endif
+                                                % if item['tableData'].get('rows'):
+                                                    % for row in item['tableData']['rows']:
+                                                        <tr>
+                                                            % for idx, col in enumerate(item['tableData'].get('columns')):
+                                                                % if row[str(idx)]['value'].startswith('?'):
+                                                                    <td>
+                                                                        <%
+                                                                            try:
+                                                                                row_answer = user_answers[index][user_answer_input_index] if user_answers else ''
+                                                                            except IndexError:
+                                                                                row_answer = ''
+                                                                        %>
+                                                                        <input type="text" name="${index}" value="${row_answer}"/>
+                                                                    </td>
+                                                                    <% user_answer_input_index += 1 %>
+                                                                % elif row[str(idx)]['type'] == 'head':
```

Comment:
```suggestion
                                                                % elif row[str(idx)].get('type') == 'head':
```

Reply:
not all cells have this key (as appeared, need to be fixed in cms part)

Reply:
Done